.github/workflows/sync-onnxrt-main.yaml

```diff
@@ -1,46 +1,52 @@
 name: Onnxruntime main weekly sync
 on:
   schedule:
-    - cron: "05 17 * * 1"
+    - cron: '27 18 * * 4'
 
 jobs:
-  runs-on: ubuntu-latest
-  sync:
+  createPullRequest:
+    name: Update and create pull request
+    runs-on: self-hosted
     steps:
-      - uses: actions/checkout@v3
-        with:
-          ref: develop
-          path: ../
+      - name: get_date
+        run: echo todays_date="$(date +'%Y-%m-%d')" >> $GITHUB_ENV
 
-  get_date:
-    steps:
-      - run: echo "::set-output name=date::$(date +'%Y-%m-%d')"
+      - name: extract_sha1 
+        run: echo onnxsha="$(git ls-remote https://github.com/microsoft/onnxruntime.git HEAD | awk '{print $1}')" >> $GITHUB_ENV
 
-  update_file:
-    needs: [sync get_date]
-    steps:
-      - run: git clone https://github.com/microsoft/onnxruntime.git && cd onnxruntime && git rev-parse HEAD >> ../test/onnx/.onnxrt-commit
+      - name: echo_sha1
+        run: echo ${{ env.onnxsha }}
 
-  Add_commit:
-    needs: update_file
-    steps:
-      - name: Add & Commit
-        uses: EndBug/add-and-commit@v9.1.1
-        with:
-          new_branch: onnxruntime-sync-${{ steps.date.outputs.date }}
-          add: ../test/onnx/.onnxrt-commit
-          message: Update Onnxruntime commit to latest release
-          default_author: github_actions
-          push: true
-  PR:
-    needs: Add_commit
-    steps:
-      - name: GitHub Action for creating Pull Requests
-        uses: devops-infra/action-pull-request@v0.5.3
+      - uses: actions/checkout@v3
+        with: 
+          ref: master
```

Comment:
Is this `master` branch name for onnx-rt ?

Reply:
No its for migrahx